docs/exts/extra_files_with_substitutions.py

```diff
@@ -25,19 +25,28 @@ def copy_docker_compose(app, exception):
     if exception or not isinstance(app.builder, builders.StandaloneHTMLBuilder):
         return
 
-    # Replace `|version|` in the docker-compose.yaml that we produce in the built docs
-    for path in app.config.html_extra_with_substituions:
+    # Replace `|version|` in the docker-compose.yaml that requires manual substitutions
+    for path in app.config.html_extra_with_substitutions:
         with open(path) as file:
-            with open(os.path.join(app.outdir, os.path.basename(path)), "w") as output:
+            with open(os.path.join(app.outdir, os.path.basename(path)), "w") as output_file:
                 for line in file:
-                    output.write(line.replace('|version|', app.config.version))
+                    output_file.write(line.replace('|version|', app.config.version))
+
+    # Replace `|version|` in the installation.html that requires manual substitutions (in links)
+    for path in app.config.manual_substitutions_in_generated_html:
+        with open(os.path.join(app.outdir, os.path.basename(path))) as input_file:
+            content = input_file.readlines()
+        with open(os.path.join(app.outdir, os.path.basename(path)), "wt") as output_file:
+            for line in content:
+                output_file.write(line.replace('|version|', app.config.version))
```

Comment:
This is coming from install.rst right? If so then Sphinx's native replacement should already handle this!

Reply:
Not really - it does not handle it when they are part of URL :(

Reply:
`Apache Airflow |version| sdist package <https://downloads.apache.org/airflow/|version|/apache-airflow-|version|-bin.tar.gz>`_ (`asc <https://downloads.apache.org/airflow/|version|/apache-airflow-|version|-bin.tar.gz.asc>`__, `sha512 <https://downloads.apache.org/airflow/|version|/apache-airflow-|version|-bin.tar.gz.sha512>`__)